### PR TITLE
Notices because of timing issue when WooCommerce Table Rate Shipping plugin is being used

### PR DESCRIPTION
**Pull request by mihaigrigori**
https://onthegosystems.myjetbrains.com/youtrack/issue/wcgcl-14

### CODE REVIEW

File: woocommerce-gateways-country-limiter.php [hunk 1]
```diff
@@ -39,8 +39,8 @@ public function __construct() {
 
 
             $this->settings = get_option( 'woocommerce_gateways_country_limiter' );
-
-            add_action( 'init', array( $this, 'init' ), 10 );
+            //set priority to 100 to make sure all gateways are loaded
+            add_action( 'init', array( $this, 'init' ), 100 );
```
Comment (andreasciamanna) [inline woocommerce-gateways-country-limiter.php]: @mihaigrigori 100 may seem arbitrary. What about using `PHP_INT_MAX`?

**Review by andreasciamanna — APPROVED** [review]
Besides the comment, I think it's fine.

**Review by andreasciamanna — APPROVED** [review]

File: woocommerce-gateways-country-limiter.php (hunk 1 above)
Comment (mihaigrigori) [reply woocommerce-gateways-country-limiter.php]: I actually only moved this from Gitlab. It was created originally by Sergey.
I agree and changed it.

@grey-rsi There wasn't any reason to have it exactly 100, right?

Comment (ghost) [reply woocommerce-gateways-country-limiter.php]: yes PHP_INT_MAX should be also fine

**Pull request merged**